.shellcheckrc

```diff
@@ -0,0 +1,3 @@
+# Following source doesn't work in most setups
+disable=SC1090
```

Comment:
We don't want to enable shellcheck for now

Reply:
I would note that you could enable trunk with shellcheck today and continue to land code without tripping any errors - that's the value of our hold-the-line feature. Basically your tech debt is your tech debt and shouldn't get in the path of forward movement. It is a nice tool to have and to understand where those potential bugs/issues are hiding.

Reply:
We've gone through shellcheck and don't want to enable all of its features, it's not a matter of not getting existing code in line (if it was, we'd just update the scripts to match what shellcheck expects)